examples/dt/dcn/service-values.yaml

```diff
@@ -196,13 +196,45 @@ data:
         replicas: 1
         type: edge
   manila:
-    enabled: false
+    enabled: true
     manilaAPI:
       customServiceConfig: |
         [DEFAULT]
         enabled_share_protocols=nfs,cephfs
+    manilaScheduler:
```

Comment:
as this is a `configMap` that points to the default control plane provided in `lib/controlplane`, I realizedwe do not need `manilaScheduler` here. Can you remove L204 andL205?

Reply:
Done